llmfoundry/tokenizers/tiktoken.py

```diff
@@ -88,6 +88,9 @@ def pickle_Encoding(enc: Encoding):
         self.add_bos_token = add_bos_token
         self.add_eos_token = add_eos_token
 
+        # Pin default ChatML Format in case HF changes defaults
+        chat_template = "{% for message in messages %}{{'<|im_start|>' + message['role'] + '\n' + message['content'] + '<|im_end|>' + '\n'}}{% endfor %}"
```

Comment:
It looks like HF implements this is a property of the tokenizer. Is there a reason you did it this way? would prefer to follow HF if there is no particular reason.

https://github.com/huggingface/transformers/blob/38e2633f80a4924bf613b0240622492beee4cfcc/src/transformers/models/llama/tokenization_llama.py#L415

Reply:
I can't get it to load the template when I do this in tests, test_chat_formatting has chat_template set to None - not too sure what I'm doing wrong

Reply:
ok fixed now